### ISSUE TITLE
Create a downloading/downloaded (but not installed) apps section

### ISSUE THREAD

**Issue by Simpath** [Iamlooker/Droid-ify#43]
if you had downloaded multiple apps, it's more likely that after sometime you will forget the names of apps.
and if notification gets cleard, It's not very convenient to find downloaded apps in available section
And even if notification isn't cleared, there is a fear of it happening

**Comment by vertigo220** [comment]
This is one of the biggest problems I'm having so far with this and Neo Store. Since the apps don't auto-update, this is a must for usability.

**Comment by Pantyhose-X** [comment]
Please try to add this feature.it is an important feature because when you download many app inside background. you have to search that apps for install. If all download apps are shown in one tab that is easier for install.

**Comment by tcely** [comment]
Downloading or downloaded apps should have a page, like favorites. Also, importantly you should be able to cancel and/or delete downloads from that page.

I also want the updates page to sort completely download updates as a group distinct from available updates that still need to be downloaded before they can be installed.

**Comment by DJCrashdummy** [comment]
absolutely, a download should have its own notification with a progress bar and a button to skip. and after finishing the download it should stay there and start the session installer if tapped.

**but** i'm not sure if these apps should have an extra section as this is a complete non-issue if you use unattended installation. :thinking:
maybe it is even possible instead of creating an additional section, to merge it with the "updates" section? like simply putting apps with downloaded APKs in the cache at the top of it (perhaps with a "dismiss" button beside the "install" button for the impatient ones which can't wait until the cache is cleared?).
yes, i know it is not really an update, but as you pressed download, it is clearly indicated that you intend to install it. FWIW call the section "updates", "tasks", "pending" or something like that.

i'm also not completely happy with this approach, but the one thing which is IMHO definitely worse, is introducing a new section which is completely useless to a big chunk of the users.
...and the one thing i would like generally, is the pinning of apps on top with already downloaded APKs. :grin:

**Comment by xeruf** [comment]
I would actually suggest incorporating them into the "updates" section.

**Comment by lazy-pupp** [comment]
maybe ->libraries->myapps=>|->dowloaded-updates->A1{->installed-updates=updateapp.apk->uninstalled-updates=updateapp.apk-|

Poslano iz aplikacije Outlook za Android<https://aka.ms/AAb9ysg>
________________________________
